common/src/main/java/com/wynntils/utils/MathUtils.java

```diff
@@ -20,4 +22,41 @@ public static float inverseLerp(float a, float b, float value) {
     public static double inverseLerp(double a, double b, double value) {
         return (value - a) / (b - a);
     }
+
+    public static int integerFromRoman(String numeral) {
```

Comment:
I was like "there must be a apache commons method to do this"... and then 5 minutes later "wat! really not?" *sigh*

Reply:
I am pretty sure we have a better version of this in legacy where it can convert anything basically.

Reply:
if we do, i didn't find it - we definitely have this one

Reply:
Ah it is for the other way around.